client/src/game.js

```diff
@@ -110,6 +137,7 @@ export default class Game {
               this.onlinePlayer[playerId].update()
             };
         };
+        this.camera.move(this.character);
```

Comment:
Maybe this method could get a name that better describes what it does. Now one could ask why the camera needs the character to move itself. If we would call the method `moveToKeepObjectInScreen` instead of just `move`. What do you think? Maybe you have a better name but when I imagine us in a few years reading this code, a more descriptive name would help us to understand the code faster.